src/main/java/org/mockito/internal/runners/RunnerFactory.java

```diff
@@ -48,7 +48,7 @@ public MockitoTestListener get() {
      */
     public InternalRunner create(Class<?> klass, Supplier<MockitoTestListener> listenerSupplier) throws InvocationTargetException {
         try {
-            return new RunnerProvider().newInstance("org.mockito.internal.runners.SilentJUnitRunner", klass, listenerSupplier);
+            return new RunnerProvider().newInstance("org.mockito.internal.runners.DefaultInternalRunner", klass, listenerSupplier);
```

Comment:
Since the runner is in the same package as the factory I don't see why it should be a String ?